src/Driver/UploadDriver.php

```diff
@@ -66,4 +69,23 @@ protected function triggerFileUploadedEvent($disk, $path, Closure $fileUploaded
 
         event(new FileUploaded($disk, $path));
     }
+
+    /**
+     * Validate an uploaded file. An exception is thrown when it is invalid.
+     *
+     * @param \Illuminate\Http\UploadedFile|null $file
+     *
+     * @throws BadRequestHttpException when given file is null.
```

Comment:
I think it would be nice to use the fully qualified name here as not all editors understand imported classes in doc blocks